src/core/importType.js

```diff
@@ -13,8 +13,9 @@ export function isAbsolute(name) {
 }
 
 export function isBuiltIn(name, settings) {
+  const baseModule = name.split('/')[0]
```

Comment:
That's a good point, thanks. Maybe we need something better than a simple split[0]. Let me try.